tideways/by-site.php

```diff
@@ -0,0 +1,18 @@
+#!/bin/env php    
+<?php 
+if(is_file('/srv/tideways.json')) {
```

Comment:
I think now that we've changed how the synced folders works, we should put this in `/srv/config` so it's in a shared folder